iamctl/pkg/utils/commonUtils.go

```diff
@@ -59,6 +61,55 @@ func getResourcePath(resourceType string) string {
 	return ""
 }
 
+func SendExportRequest(resourceId, fileType, resourceType string, excludeSecrets bool) (resp *http.Response, err error) {
+
+	var reqUrl string
+	if resourceType == APPLICATIONS {
+		reqUrl = SERVER_CONFIGS.ServerUrl + "/t/" + SERVER_CONFIGS.TenantDomain + "/api/server/v1/" + getResourcePath(resourceType) + "/" + resourceId + "/exportFile"
+	} else {
+		reqUrl = SERVER_CONFIGS.ServerUrl + "/t/" + SERVER_CONFIGS.TenantDomain + "/api/server/v1/" + getResourcePath(resourceType) + "/" + resourceId + "/export"
+	}
```

Comment:
let's extract this out to a separate method. something like buildRequestUrl

Reply:
Added a new method `buildRequestUrl` that returns the request url for each request.